core/validation/failures.py

```diff
@@ -17,6 +17,44 @@ def __str__(self):
         """Abstract method to get the string representation of the failure."""
 
 
+INTERNAL_TABLE_TO_FORM_LOCATION = {
+    "Project Details": ("Project Admin", "Project Details"),
+    "Project Progress": ("Programme Progress", "Projects Progress Summary"),
+    "Funding": ("Funding Profiles", "Project Funding Profiles"),
+    "Outcome_Data": ("Outcomes", "Outcome Indicators (excluding footfall)"),
+    "RiskRegister": ("Risk Register", "Programme / Project Risks"),
```

Comment:
Can we reliably map 1 section per sheet like this, what if a sheet has more than one section to consider?

Reply:
Good point, this will work for this UC message as there is only one section per sheet that it can occur on. Makes more sense to add the section to the column mapping instead.